test/plugins/test_threegpp/ietf-netconf-acm.yang

```diff
@@ -0,0 +1,385 @@
+module ietf-netconf-acm {
```

Comment:
Hi, I am not quite sure if this file is needed or not, but this file already exists in the path `module/ietf/`

Reply:
I removed the IETF Yang modules are used for testing, referencing them instead from modules/ietf.
Please merge